docs/index.md

```diff
@@ -1,3 +1,150 @@
-{%
-    include-markdown "../README.md"
-%}
+# Project Status Badges
+
+[![Code Quality](https://github.com/8ball030/auto_dev/actions/workflows/common_check.yaml/badge.svg)](https://github.com/8ball030/auto_dev/actions/workflows/common_check.yaml)
+
+
+# Autonomy Dev
+
+Tooling to speed up open-autonomy development.
+
+
+## TLDR
+
+```bash
+# install 
+pip install autonomy-dev[all]
+```
```

Comment:
this entire page is a duplication of the readme right? 
Can we instead reference the read me to avoid duplication which will immeidately lead to issues